### ISSUE TITLE
Implement asymmetric public/private encryption

### ISSUE THREAD

**Issue by michaelstepner** [gaubert/gmvault#178]
Since the encryption key is symmetric and must be on the same machine as the emails being encrypted, encryption provides no additional security if the machine were compromised.

It seems to me (please correct me if I'm mistaken) that the only additional security offered by encryption is that if I were to back up the database to a remote server without backing up the encryption key, the encryption would secure my data on the remote server.

I think it would be very valuable to have true public/private key encryption in GMVault.  @tripplet and @nothsa made the same request as comments on issue https://github.com/gaubert/gmvault/issues/62

**Comment by gaubert** [comment]
yes I agree to the fact that the public key could be used to encrypt the mails and only the private key could then decrypt them but this requires a different management and makes it more complicated.
I will see what can be done but it isn't a high priority issue for the moment.

**Comment by michaelstepner** [comment]
Thanks Guillaume.  I understand it's not at the top of your priorities, but I hope to see asymmetric encryption in a future version!
